plugins/modules/vsphere_copy.py

```diff
@@ -125,7 +146,8 @@ def main():
         datacenter=dict(required=False),
         datastore=dict(required=True),
         path=dict(required=True, aliases=['dest'], type='str'),
-        timeout=dict(default=10, type='int'))
+        timeout=dict(default=10, type='int'),
+        diskformat=dict(required=False, type='str'))
```

Comment:
I suggest replace
```diskformat=dict(required=False, type='str'))```
on
```diskformat=dict(required=False, type='str', choices=['streamVmdk']))``` and add it to documentation. This way we will fix the possible values ​​and will not need to handle the input error.